integrations/clickup/.port/spec.yaml

```diff
@@ -0,0 +1,31 @@
+type: clickup
+description: ClickUp integration for Port Ocean
+icon: Clickup
+features:
+  - type: exporter
+    section: Project Management
+    resources:
+      - kind: issue
+      - kind: project
+      - kind: space
+      - kind: team
+configurations:
+  - name: clickupPersonalToken
+    description: The ClickUp personal token used for authentication. To create a personal token, see the <a href="https://clickup.com/api/developer-portal/authentication#personal-token">ClickUp documentation</a>
+    required: true
+    type: string
+    sensitive: true
+  - name: clickupBaseUrl
+    description: The Clickup Base URL. Defaults to https://api.clickup.com when not specified
+    type: url
```

Comment:
href this as well and make sure it opens in another tab, see other integrations for reference

Reply:
Done